runtime/android/core_internal/BUILD.gn

```diff
@@ -0,0 +1,215 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/config/android/rules.gni")
+import("//xwalk/build/version.gni")
+import("//xwalk/build/xwalk.gni")
+
+android_library("xwalk_core_internal_java") {
+  deps = [
+    ":xwalk_core_internal_java_resources",
+    "//base:base_java",
+    "//components/navigation_interception/android:navigation_interception_java",
+    "//components/web_contents_delegate_android:web_contents_delegate_android_java",
+    "//content/public/android:content_java",
+    "//media/base/android:media_java",
+    "//net/android:net_java",
+    "//ui/android:ui_java",
+    "//xwalk/extensions/android:xwalk_core_extensions_java",
+  ]
+  srcjar_deps = [ ":bridge" ]
```

Comment:
I found `android_library` doesn't allow user to insert "action" and "copy" into its deps. The only way I found is to use the `srcjars_deps `.  If I change the code as below:
```
android_library("xwalk_core_internal_java") {
  deps = [ 
    ":xwalk_core_reflection_layer_java_gen",  #outputs "$reflection_gen_dir/bridge/bridge.srcjar"
    ......
  ]
  srcjars = [ "$reflection_gen_dir/bridge/bridge.srcjar" ]
  ......
}

```
"gn gen" complains:
```
ERROR Input to target not generated by a dependency.
The file:
  //out/an/gen/xwalk_core_reflection_layer/bridge/bridge.srcjar
is listed as an input or source for the target:
  //xwalk/runtime/android/core_internal:xwalk_core_internal_java__compile_java__javac
but this file was not generated by any dependencies of the target. The target
that generates the file is:
  //xwalk/runtime/android/core_internal:xwalk_core_reflection_layer_java_gen

If you have generated inputs, there needs to be a dependency path between the
two targets in addition to just listing the files. For indirect dependencies,
the intermediate ones must be public_deps. data_deps don't count since they're
only runtime dependencies. If you think a dependency chain exists, it might be
because the chain is private. Try "gn path" to analyze.
```
Then I run the "gn path", it shows:
```
gn path out/an //xwalk/runtime/android/core_internal:xwalk_core_reflection_layer_java_gen //xwalk/runtime/android/core_internal:xwalk_core_internal_java__compile_java__javac
//xwalk/runtime/android/core_internal:xwalk_core_internal_java__compile_java__javac --[private]-->
//xwalk/runtime/android/core_internal:xwalk_core_internal_java__build_config --[private]-->
//xwalk/runtime/android/core_internal:xwalk_core_reflection_layer_java_gen

1 non-data path found. It is not public.
heke@heke-workstation:~/master/crosswalk-src/src$ gn path out/an //xwalk/runtime/android/core_internal:xwalk_core_reflection_layer_java_gen //xwalk/runtime/android/core_internal:xwalk_core_internal_java__compile_java__javac
//xwalk/runtime/android/core_internal:xwalk_core_internal_java__compile_java__javac --[private]-->
//xwalk/runtime/android/core_internal:xwalk_core_internal_java__build_config --[private]-->
//xwalk/runtime/android/core_internal:xwalk_core_reflection_layer_java_gen

1 non-data path found. It is not public.
```